ajax/libs/jquery.ime/1.0.0/jquery.ime.css

```diff
@@ -0,0 +1,217 @@
+.imeselector {
```

Comment:
The file structure should be 
```
├── css
│   └── jquery.ime.css
```
Many thanks.

Reply:
1) I opened an issue. lets wait for the author answer
2) We need this files.. See the demo pages. It is used for show right bottom corner querty image
3) The files were updated from the stream with wget